doozerlib/distgit.py

```diff
@@ -935,6 +937,17 @@ def _extract_container_build_logs(self, task_id):
         except OSError as e:
             self.logger.warning("Exception while trying to extract build logs in {}: {}".format(logs_dir, e))
 
+    def _add_missing_pkgs(self, pkg_name):
+        """
+        add missing packages to runtime.missing_pkgs set and print it out for
+        `brew tag-build` eg: from `rhaos-4.0-rhel-7-candidate` to `rhaos-4.1-rhel-7-candidate` manually
+        :param pkg_name: Missing packages like: No package (.*) available
+        """
+        latest = self.metadata.exec_latest_build_info(pkg_name)
+        if latest.startswith(pkg_name):
+            with self.runtime.mutex:
+                self.runtime.missing_pkgs.add(latest)
```

Comment:
In addition to what was missing, could the output indicate which image build wanted it? Helpful context.

Reply:
Are u suggest change a name of `missing-pkgs`?

Reply:
The name is fine. It's just that it's going to say something like
```
Missing packages:
foo
bar
```

And it would be nice if it was more like
```
Missing packages:
openshift-enterprise-base image is missing foo
ansible-service-broker image is missing bar
```
For now each entry in missing_pkgs would be one of those lines.
```suggestion
    self.runtime.missing_pkgs.add("{} image is missing package {}".format(self.metadata.name, pkg_name))
```